### PR TITLE
Updates bee lore

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
tweak: Updates bee lore
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/bees.dm [hunk 1]
```diff
@@ -12,7 +12,4568 @@
...
+  
+I hear they put the roaches in motels.
+That doesn't sound so bad.
+
+  
+Adam, they check in,
+but they don't check out!
+
+  
+Oh, my.
```
Comment (JJRcop) [inline code/modules/mob/living/simple_animal/hostile/bees.dm]: this

**Comment by Xhuis** [comment]
🐝

**Comment by Thunder12345** [comment]
🐝

**Comment by Kraseo** [comment]
🐝

**Pull request closed without merging**